splinterd/api/static/openapi.yml

```diff
@@ -469,3 +608,34 @@ components:
         prev: /nodes?offset=0&limit=10
         next: /nodes?offset=20&limit=10
         last: /nodes?offset=40&limit=10
+
+    Notification:
```

Comment:
How do we know if the notification is read or not? If that functionality is part of the user resource, that should be added in this design.

Reply:
updated